blazor/file-manager/pagination.md

```diff
@@ -0,0 +1,192 @@
+
+The following example demonstrates how to change the page size of a File Manager using the `PageSize` property.
+
+````cshtml
+@using Syncfusion.Blazor.FileManager;
+@using Syncfusion.Blazor.Navigations;
+
+<SfFileManager  TValue="FileManagerDirectoryContent" AllowPaging="true">
+    <FileManagerAjaxSettings Url="https://ej2-aspcore-service.azurewebsites.net/api/Virtualization/FileOperations"
+                             UploadUrl="https://ej2-aspcore-service.azurewebsites.net/api/Virtualization/Upload"
+                             DownloadUrl="https://ej2-aspcore-service.azurewebsites.net/api/Virtualization/Download"
+                             GetImageUrl="https://ej2-aspcore-service.azurewebsites.net/api/Virtualization/GetImage">
+    </FileManagerAjaxSettings>
+    <FileManagerPageSettings PageSize="3"></FileManagerPageSettings>
+</SfFileManager>
+````
+Below is a screenshot illustrating the `PageSize` property in the File Manager component.
+
+![Pagination in Blazor FileManager](images/blazor-filemanager-pagesize.png)
+
+### Change the page count
+
+The `NumericItemsCount` property is used to control the number of numeric buttons displayed in the pager when pagination is enabled.
+ ````cshtml
+ @using Syncfusion.Blazor.FileManager;
+@using Syncfusion.Blazor.Navigations;
+<SfFileManager TValue="FileManagerDirectoryContent" AllowPaging="true" >
+    <FileManagerAjaxSettings Url="https://ej2-aspcore-service.azurewebsites.net/api/Virtualization/FileOperations"
+                             UploadUrl="https://ej2-aspcore-service.azurewebsites.net/api/Virtualization/Upload"
+                             DownloadUrl="https://ej2-aspcore-service.azurewebsites.net/api/Virtualization/Download"
+                             GetImageUrl="https://ej2-aspcore-service.azurewebsites.net/api/Virtualization/GetImage">
+    </FileManagerAjaxSettings>
+    <FileManagerEvents TValue="FileManagerDirectoryContent" PageChanging="OnChanging" PageChanged="OnChanged"></FileManagerEvents>
+    <FileManagerPageSettings NumericItemsCount="5"></FileManagerPageSettings>
+</SfFileManager>
+ ````
+
+Below is a screenshot illustrating the `NumericItemCount` property in the File Manager component.
+
+![Pagination in Blazor FileManager](images/blazor-filemanager-numericitemcount.png)
+
+### Change the current page
+
+The Syncfusion File Manager allows you to change the currently displayed page, which can be particularly useful when you need to navigate through different pages of data either upon the initial rendering of the file manager or update the displayed page based on interactions or specific conditions. The default value of **CurrentPage** property is 1.
```

Comment:
Use same component casing @saravananj-sf4375

Reply:
modified the changed and updated the PR.